File: practise questions/2.c
// program to add every element which is present at even position

#include <stdio.h>

void evenposition();

int main()
{
    evenposition();
    return 0;
}
void evenposition()
{
    int sum = 0;
    int arr[9] = {10, 20, 30, 40, 50, 60, 70, 80, 90};
    for (int i = 0; i < 9; i = i + 2)
    {
        if (i % 2 == 0)
        {
            sum = sum + arr[i];
        }
    }
    printf("sum is %d", sum);
}

File: Java/Assignment 8/2.java
public class MyBinarySearch {
	public void MySearch(int arr[],int no) {
		int mid;
		int start;
		int end;
		start=0;
		end=arr.length-1;
		
		while (end>=start) {
			mid=start+end;
			if (arr[mid]==no) {
				System.out.println("position of element is"+mid);
				return;
			}
			else if (arr[mid]>no) {
				end=mid-1;
			}
			else if (arr[mid]<no) {
				start=mid+1;
			}
			
		}
		
			System.out.println("no no present");
	}
}
